Configuration/Geometry/python/dict2023Geometry.py

```diff
@@ -562,19 +528,13 @@
 allDicts = [ commonDict, trackerDict, caloDict, muonDict, forwardDict, timingDict ]
 
 detectorVersionDict = {
-    ("O1","T3","C1","M1","F1","I1") : "D7",
     ("O1","T4","C1","M1","F1","I1") : "D10",
-    ("O1","T3","C2","M1","F1","I1") : "D4",
-    ("O1","T3","C2","M1","F1","I2") : "D8",
-    ("O1","T3","C1","M2","F1","I1") : "D9",
     ("O1","T5","C2","M1","F1","I1") : "D11",
-    ("O2","T3","C2","M2","F2","I1") : "D12",
-    ("O1","T3","C3","M2","F1","I1") : "D13",
     ("O2","T5","C2","M2","F2","I1") : "D14",
-    ("O2","T3","C3","M2","F2","I1") : "D15",
     ("O1","T5","C3","M2","F1","I1") : "D16",
     ("O2","T5","C3","M2","F2","I1") : "D17",
+    ("O1","T5","C2","M1","F1","I2") : "D18",

```

Comment:
@boudoul - should it be O2 since @kpedro88 mentioned at sim meeting that the detailed cavern poses no problems and will be used for TDR samples?